pkg/model/core/decodable.go

```diff
@@ -23,33 +23,42 @@ import (
 type Decodable[T any] struct {
 	Value T
 
-	// TODO: Considering that these parameters are at most
-	// aes.BlockSize bytes in size, would it make sense to replace
-	// this with a fixed size array?
-	decodingParameters string
+	// Lets use a fixed-size array for decoding parameters, so Decodable[] can be marshalled, and used as a key in a map.
```

Comment:
s/Lets/Let's/
s/marshalled/marshaled/

Can you please wrap this comment to <80 columns? Thanks!